newsroom/templates/login.html

```diff
@@ -42,5 +42,10 @@ <h5 class="mb-0">{{ gettext("Login") }}</h5>
             </div>
         </div>
     </div>
+    <div class="row">
```

Comment:
looks like this should be part of some layout page

Reply:
That requires two layouts, do you want me to create as there are two inner divs?